app/controllers/DigitalSubscription.scala

```diff
@@ -106,14 +107,20 @@ class DigitalSubscription(
 
   def create: Action[CreateSupportWorkersRequest] =
     authenticatedAction(recurringIdentityClientId).async(circe.json[CreateSupportWorkersRequest]) {
-      implicit request =>
+      implicit request: AuthRequest[CreateSupportWorkersRequest] =>
         val billingPeriod = request.body.product.billingPeriod
         SafeLogger.info(s"[${request.uuid}] User ${request.user.id} is attempting to create a new $billingPeriod digital subscription")
-        val result = for {
-          user <- identityService.getUser(request.user)
-          statusResponse <- client.createSubscription(request, createUser(user, request.body), request.uuid).leftMap(_.toString)
-        } yield statusResponse
-        respondToClient(result, request.body.product.billingPeriod)
+
+        if (validationPasses(request.body)) {
```

Comment:
This `if/else` is ugly and I'm currently thinking of how to make it less ugly, so if you have any suggestions I am all 👂